.pre-commit-config.yaml

```diff
@@ -65,3 +65,11 @@ repos:
         language: golang
         require_serial: true
         pass_filenames: false
+      - id: golangci-lint
```

Comment:
Why are there two entries for golangci-lint?

Reply:
Because each one has a different `entry`: `golangci-lint run --new-from-rev HEAD --fix` and `golangci-lint fmt`. I think you can have multiple entries for one hook ID but then you wouldn't be able to tell which command failed.

Reply:
I may be misunderstanding, but wouldn't the second entry be more comprehensive than the first and thus making the first unnecessary?

Reply:
I believe that `golangci-lint run` runs the [linters](https://github.com/cmdevoto/pipelines/blob/gofmt-changes/.golangci.yaml#L12-L26) while `golangci-lint fmt` runs the [formatters](https://github.com/cmdevoto/pipelines/blob/gofmt-changes/.golangci.yaml#L28-L32)

Reply:
Got it, makes sense! I missed the run vs fmt distinction.